demos/common/python/openvino/model_zoo/model_api/models/model.py

```diff
@@ -103,41 +161,60 @@ def load_config(self, config):
                 self.logger.warning(f'The parameter "{name}" not found in {self.__model__} wrapper, will be omitted')
 
     def raise_error(self, message):
-        '''Raises the WrapperError
+        '''Raises the WrapperError.
+
         Args:
-            message: error message
+            message (str): error message to be shown in the following format:
+              "WrapperName: message"
         '''
         raise WrapperError(self.__model__, message)
 
     def preprocess(self, inputs):
-        '''Interface for preprocess method
+        '''Interface for preprocess method.
+
         Args:
-            inputs: raw input data, data types are defined by concrete model
+            inputs: raw input data, the data type is defined by wrapper
+
         Returns:
-            - The preprocessed data ready for inference
-            - The metadata, which could be used in postprocessing
+            - the preprocessed data which is submitted to the model for inference
+                and has the following format:
+                {
+                    'input_layer_name_1': data_1,
+                    'input_layer_name_2': data_2,
+                    ...
+                }
+            - the input metadata, which might be used in `postprocess` method
         '''
         raise NotImplementedError
 
     def postprocess(self, outputs, meta):
-        '''Interface for postrpocess metod
+        '''Interface for postprocess method.
+
         Args:
-            outputs: the model outputs  as dict with `name: tensor` data
-            meta: the metadata from the `preprocess` method results
+            outputs (dict): model raw output in the following format:
+                {
+                    'output_layer_name_1': raw_result_1,
+                    'output_layer_name_2': raw_result_2,
+                    ...
+                }
+            meta (dict): the input metadata obtained from `preprocess` method
+
         Returns:
-            Postrocessed data in format accoding to conrete model
+            - post-processed data in the format defined by wrapper
```

Comment:
```suggestion
            - postprocessed data in the format defined by wrapper
```
since `preprocecess` word you've spelled without hyphen, so let's here be the same.
btw both variants  are acceptable as I see in google)